src/module_wrap.cc

```diff
@@ -375,7 +375,9 @@ void ModuleWrap::Evaluate(const FunctionCallbackInfo<Value>& args) {
     return;
   }
 
-  args.GetReturnValue().Set(result.ToLocalChecked());
+  if (env->options()->harmony_top_level_await) {
+    args.GetReturnValue().Set(result.ToLocalChecked());
+  }
```

Comment:
Could you leave a comment explaining this change?

Reply:
I am not sure if it is clever to check for the harmony top-level flag to choose to apply this we need a other solution as this should get unflagged at a later point anyway